pom.xml

```diff
@@ -1533,6 +1533,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-invoker-plugin</artifactId>
+                        <version>3.2.2</version>
```

Comment:
starting with this version `streamLogsOnFailures` has been added (to help with logging output for CI usage)

Reply:
Good option!